README.md

```diff
@@ -1,45 +1,49 @@
+# Audacity
+
+![Audacity](images/AudacityLogo.png)
+
 Audacity(R): free, open source, cross-platform audio software for 
-multi-track recording and editing: https://www.audacityteam.org/ . 
+multi-track recording and editing: <https://www.audacityteam.org>
 
 User support is provided on Audacity Forum:
-https://forum.audacityteam.org/ .
+<ttps://forum.audacityteam.org>.
 
 We welcome feedback on Audacity, suggestions for new or improved features, 
-and bug reports. Please visit https://forum.audacityteam.org/viewforum.php?f=25 .
+and bug reports. Please visit <https://forum.audacityteam.org/viewforum.php?f=25>.
 
 Audacity is copyright (c) 1999-2018 by Audacity Team. This copyright 
 notice applies to all documents in the Audacity source code archive, 
 except as otherwise noted (mostly in the lib-src subdirectories). 
 "Audacity" is a registered trademark of Dominic Mazzoni. 
 
 The Audacity documentation is licensed under the Creative Commons
-Attribution 3.0 license: https://creativecommons.org/licenses/by/3.0/legalcode .
+Attribution 3.0 license: <https://creativecommons.org/licenses/by/3.0/legalcode>.
 
 Compilation instructions for Audacity are provided in the source code:
-* Windows: win\build.txt
-* macOS: mac/build.txt
-* GNU/Linux: linux/build.txt 
+* Windows: `win\build.txt`
+* macOS: `mac/build.txt`
+* GNU/Linux: `linux/build.txt`
 
 You can ask for help with compilation problems at:
-https://forum.audacityteam.org/viewforum.php?f=19 .
+<https://forum.audacityteam.org/viewforum.php?f=19>.
 
 If you want to suggest some simple text change in our code, please submit a 
-pull request on https://github.com/audacity/audacity/pulls . It's usually 
+pull request on <https://github.com/audacity/audacity/pulls>. It's usually 
 best to discuss functional code changes with us first on audacity-devel: 
-https://lists.sourceforge.net/lists/listinfo/audacity-devel . 
+<https://lists.sourceforge.net/lists/listinfo/audacity-devel>.
 
 Version 2.3.0
 
 Contents of this README:
 
 1.  Licensing
-2.  Changes since version 2.2.2
-3.  Known Issues at Release
-4.  Source Code, Libraries and Additional Copyright Information
+1.  Changes since version 2.2.2
+1.  Known Issues at Release
+1.  Source Code, Libraries and Additional Copyright Information
 
```

Comment:
strange that they're all numbered "1" ?

Reply:
this is a convenient way markdown provide to auto number the list. We could keep 1. 2. 3. 4 for people that don't have markdown viewer thus.

Reply:
@MartinDelille: do you mean that it's only wrongly displayed here in the commit, but will auto-magically display 1., 2.,... when merged?

Reply:
No only in the viewer. Try to copy/paste the code here: http://markdownlivepreview.com/

Reply:
OK